### PR TITLE
[ticket/14234] Use replacement variables instead of references in events

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-14234

### CODE REVIEW

File: phpBB/includes/functions_acp.php [hunk 1]
```diff
@@ -392,14 +393,17 @@ function build_cfg_template($tpl_type, $key, &$new, $config_key, $vars)
...
 	* @var	array	vars		Array with the options for the config
 	* @var	string	tpl			The resulting html code we display
 	* @since 3.1.0-a1
+	* @change 3.1.7-RC1 Replaced new with new_ary
 	*/
-	$vars = array('tpl_type', 'key', 'new', 'name', 'vars', 'tpl');
+	$vars = array('tpl_type', 'key', 'new_ary', 'name', 'vars', 'tpl');
 	extract($phpbb_dispatcher->trigger_event('core.build_config_template', compact($vars)));
+	$new = $new_ary;
+	unset($new);
```
Comment (rxu) [inline phpBB/includes/functions_acp.php]: ```unset($new_ary)```?

File: phpBB/includes/acp/acp_forums.php [hunk 2]
```diff
@@ -950,17 +950,21 @@ function update_forum_data(&$forum_data)
...
 		/**
 		* Validate the forum data before we create/update the forum
 		*
 		* @event core.acp_manage_forums_validate_data
-		* @var	array	forum_data	Array with new forum data
+		* @var	array	forum_data_ary	Array with new forum data
 		* @var	array	errors		Array of errors, should be strings and not
 		*							language key.
 		* @since 3.1.0-a1
+		* @change 3.1.7-RC1 Replaced forum_data with forum_data_ary
```
Comment (rxu) [inline phpBB/includes/acp/acp_forums.php]: If we aren't going to merge this into 3.1.x, this probably should be ```3.2.0-a1```.

Comment (marc1706) [inline phpBB/includes/acp/acp_forums.php]: I noticed while working on other events :)

**Comment by Nicofuma** [comment]
That's good for a first step but we need to get ride of this references when this part will be refactored

**Comment by marc1706** [comment]
@Nicofuma turns out the changes to the php-fpm config didn't help. I guess I'll just remove them from this PR then.

**Pull request merged**